innopoints/views/random.py

```diff
@@ -0,0 +1,35 @@
+"""Return a random page.
+
+- GET /random_page
+"""
+
+import random
+
+from flask import jsonify
+
+from innopoints.blueprints import api
+from innopoints.extensions import db
+from innopoints.models import Project, Product
+
+
+PAGES = [
+    '/',
+    '/projects',
+    '/products',
+    '/profile',
+    '/dashboard',
+    Product,
+    Project,
+]
+
+@api.route('/random_page')
+def get_random_page():
+    """Return a random URL on the frontend."""
+    random_page = random.choice(PAGES)
+    if isinstance(random_page, str):
+        return jsonify(url=random_page)
```

Comment:
Why does it need to be wrapped in an object when it is just one property? Isn't a string valid JSON (just like the number when updating balance)?

Reply:
Just because this looks cleaner. Again, really doesn't matter